web/core/components/workspace-notifications/sidebar/header/options/menu-option/root.tsx

```
@@ -4,6 +4,7 @@ import { FC, ReactNode } from "react";
 import { observer } from "mobx-react";
 import { Check, CheckCircle, Clock } from "lucide-react";
 import { TNotificationFilter } from "@plane/types";
+import { useTranslation } from "@plane/i18n";
 import { ArchiveIcon, PopoverMenu } from "@plane/ui";
 import { useTranslation } from "@plane/i18n";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useTranslation'. Consider to delete it or rename it.

'useTranslation' is defined here:


(lint/suspicious/noRedeclare)